docs/getting_started/advanced_setup_guides.rst

```diff
@@ -0,0 +1,273 @@
+
+Once the profile is created (a new entry should be appear in file ``~/.aws/config``\ ), you can activate it via setting environment variable:
+
+.. code-block:: shell
+
+   export AWS_PROFILE=rubrix
+
+Create docker machine (aws)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: shell
+
+   docker-machine create --driver amazonec2 \
+   --amazonec2-root-size 60 \
+   --amazonec2-instance-type t2.large \
+   --amazonec2-open-port 80 \
+   --amazonec2-ami ami-0b541372 \
+   --amazonec2-region eu-west-1 \
+   rubrix-aws
+
+Available ami depends on region. The provided ami is available for eu-west regions
+
+Verify machine creation
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: shell
+
+   $>docker-machine ls
+
+   NAME                   ACTIVE   DRIVER      STATE     URL                        SWARM   DOCKER     ERRORS
+   rubrix-aws             -        amazonec2   Running   tcp://52.213.178.33:2376           v20.10.7
+
+Save asigned machine ip
+^^^^^^^^^^^^^^^^^^^^^^^
+
+In our case, the assigned ip is ``52.213.178.33``
+
+Connect to remote docker machine
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable the connection between the local docker client and the remote daemon, we must type following command:
+
+.. code-block:: shell
+
+   eval $(docker-machine env rubrix-aws)
+
+Define a docker-compose.yaml
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: yaml
+
+   # docker-compose.yaml
+   version: "3"
+
+   services:
+     rubrix:
+       image: recognai/rubrix
+       ports:
+         - "80:80"
+       environment:
+         ELASTICSEARCH: <elasticsearch-host_and_port>
+       restart: unless-stopped
+
+Pull image
+^^^^^^^^^^
+
+.. code-block:: shell
+
+   docker-compose pull
+
+Launch docker container
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: shell
+
+   docker-compose up -d
+
+Accessing to rubrix
+^^^^^^^^^^^^^^^^^^^
+
+In our case http://52.213.178.33
+
+
+.. _users-management:
+
+Users management
+----------------
+
+The Rubrix server allows you to manage various users, which helps you to keep track of the annotation agents.
+
+Default user credentials
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- username: ``rubrix``
+- password: ``1234``
+- api key: ``rubrix.apikey`` (configured automatically by the client)
+
+
+Override user api key
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: shell
+
+    export RUBRIX_LOCAL_AUTH_DEFAULT_APIKEY=new-apikey
+
+
+Override user password
+^^^^^^^^^^^^^^^^^^^^^^
+
+You must set an environment variable that contains an already hashed password.
+
+Generate the hashed password:
+
+.. code-block:: shell
+
+   %> htpasswd -nbB "" my-new-password
+   :$2y$05$T5mHt/TfRHPPYwbeN2.q7e11QqhgvsHbhvQQ1c/pdap.xPZM2axje
+
+then, configure the environment variable omitting the first ``:`` character:
+
+.. code-block:: shell
+
+    export RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD="<generated_user_password>"
+
+
+Create a user db file
```

Comment:
All this section still seems a series of steps to follow but I think is not the case. For example, you dont need to override the default user name if you want to add more users or you don t need to use the docker compose if you launch the python server directly. I'd recommend more explicit headings, like  how to add new users, how to override the default user, and then subsections: launching the server, using docker, etc.

Reply:
Done, I restructured this part a bit: https://docs.rubrix.ml/en/feat-revise_docs/getting_started/advanced_setup_guides.html#users-management